docs/reST/ref/display.rst

```diff
@@ -172,6 +152,32 @@ required).
 
    .. versionadded:: 2.0.0 ``SCALED``, ``SHOWN`` and ``HIDDEN``
 
+   When requesting fullscreen display modes, sometimes an exact match for the
+   requested size cannot be made. In these situations pygame will select
+   the closest compatible match. The returned surface will still always match
+   the requested size.
+
+   On high resolution displays(4k, 1080p) and tiny graphics games (640x480)
+   show up very small so that they are unplayable. SCALED scales up the window
+   for you. The game thinks it's a 640x480 window, but really it can be bigger.
+   Mouse events are scaled for you, so your game doesn't need to do it. Note
+   that SCALED is considered an experimental API and may change in future
+   releases.
+
+   The ``depth`` argument represents the number of bits to use for color. It is 
+   usually best to not pass the depth argument. It will default to the
+   best and fastest color depth for the system. If your game requires a
```

Comment:
I think documentation overhauls like this are better suited to a separate PR. IMO, each PR should be fairly focused on what it's purpose is and not cover too many changes outside of that particular scope.

Reply:
Thank you for your review. I will fix that.